falcosidekick/values.yaml

```diff
@@ -86,8 +86,8 @@ config:
     cacertfile: "/etc/certs/server/ca.crt"
     # -- port to serve http server serving selected endpoints
     notlsport: 2810
-    # -- a comma separated list of endpoints, if not empty, a separate http server will be deployed for the specified endpoints
-    notlspaths: ""
+    # -- a comma separated list of endpoints, if not empty, a separate http server will be deployed for the specified endpoints. Ping endpoint needs to be notls to be able to check the health of the pod.
```

Comment:
it's a non-sense, as you set a default value, it will never be empty

Reply:
If mTLS is enabled, Kubernetes won't be able to get a 200 response when performing a readiness/liveness check, and the pod will be restarted until changed here, or specifying the nomtls port in the service.

Reply:
I know, I'm only saying the description is not accurate

Reply:
Changed!